semantic_conventions/http-common.yaml

```diff
@@ -80,8 +79,6 @@ groups:
           SHOULD NOT be set if only IP address is available and capturing name would require a reverse DNS lookup.
 
       - ref: server.port
-        requirement_level:
```

Comment:
This attribute is actually not clear to me and should definitely be clarified

If this is set to the actual port, then it seems perfectly fine on the server side (as there is a bounded number of listen ports, usually only two (http + https)).
If this is the "logical port" e.g. from the Host header, which it should probably be, matching the server.address, then we need to update this specification to say so.

I think the domain + port are often treated as belonging together when it comes to virtual host matching (which we talk about in the definition of server.address

Reply:
> In the end, there should always be the possibility to compute a full URL for any monitored HTTP request.

not arguing, but can you help me understand why is it needed (esp on metrics) and if we can fall back to `service.name` as a last resort.

reconstructing a URL can never be reliable (address and port come from the Host header, modified by reverse proxies), the scheme is subject to SSL termination, path can be a result of URL rewrites. What benefit user get from a reconstructed   URL which overall does not precisely describe anything specific?

Reply:
> This attribute is actually not clear to me and should definitely be clarified

there is a note below that clarifies it. 
```
          Determined by using the first of the following that applies

          - Port identifier of the [primary server host](/specification/trace/semantic_conventions/http.md#http-server-definitions) of the matched virtual host.
          - Port identifier of the [request target](https://www.rfc-editor.org/rfc/rfc9110.html#target.resource)
            if it's sent in absolute-form.
          - Port identifier of the `Host` header
```

Regardless of the cardinality, if we don't set `server.address`, `port` alone is not helpful. 
Since we assume `address:port` combination is static within one resource attribute combination, there is no real point in setting them on metrics.

Reply:
> reconstructing a URL can never be reliable

That is true.

>  (esp on metrics)

This change affects all signals though.

Reply:
spans still have server address and port at recommended level, if you believe it's not the case, please let me know what can I do to make it more explicit

Reply:
> The URL seems like the most basic thing to know about any HTTP request.

about client HTTP request. incoming requests don't have a full URL ready.